### DISCUSSION TITLE
Is it possible to use Supabase with Javascript ? If so, how ?

### DISCUSSION THREAD

**Discussion by dolphin007x** [supabase/supabase#1653] [Questions] [answered]
Hello !! I just found out about Supabase a couple of days ago, and I'm REALLY excited to use it. 
## Is there any way by which I can use it with Javascript ? 

```If someone knows anything about this, please do respond . I'll get back to you ASAP. ```

**Comment by kiwicopple** [comment, marked as answer]
Hi @Adatta1276 , yes that's a large part of Supabase :)

You can find the JS docs here: https://supabase.io/docs/reference/javascript/installing
